torch/_functorch/_aot_autograd/runtime_wrappers.py

```diff
@@ -69,10 +69,15 @@ def create_runtime_wrapper(
     keep_input_mutations: bool,
     disable_amp: bool,
 ):
+    num_tokens = len(runtime_metadata.tokens)
+
     if not hasattr(compiled_fn, "_boxed_call"):
         compiled_fn = make_boxed_func(compiled_fn)
 
     def runtime_wrapper(*args):
+        # Pass in effect tokens
+        args = (*[torch.tensor([])] * num_tokens, *args)
```

Comment:
It would be nice to have a `Note [Side-Effectful Tokens in AOTAutograd]` comment somewhere, where you can:

(1) describe a bit more what aot autograd does here (add extra "effect token" tensor args as inputs/outputs in the graph

(2) Link the comment in the various sections of AOTAutograd that require special handling, so someone can go back and easily find all the spots (the runtime section here, and in schemas.py / collect_metadata_analysis.py)